resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java

```diff
@@ -13,47 +14,156 @@
 @Component
  public class TemplateUtil {
 
-    public Map<String, String> getAckTemplateVariablesForCredentialShare(String eventId){
-        return Collections.emptyMap();
+    public Map<String, String> getAckTemplateVariablesForCredentialShare(String eventId) {
+        Map<String, String> templateVariables = new HashMap<>();
+        templateVariables.put("eventId", eventId);
+        templateVariables.put("eventType", "credential-share");
+        templateVariables.put("purpose", "credential-share");
+        templateVariables.put("eventStatus", "success");
```

Comment:
The values are to be calculated from the request/session. For all the features..

Reply:
Yes Fixed for these type, i am deciding values in service class only taking variable name from template variable method in TemplateUtil class.